opencart 1.5.6.4/upload/admin/controller/module/smaily_for_opencart.php

```diff
@@ -180,6 +206,22 @@ public function index() {
           ),
         );
 
+        // Display selected layout module position as such.
+        $this->data['module_position_selected'] = array(
```

Comment:
`_selected` suffix does not make sense in this case. You have none of the elements in the array in "selected" state.

Reply:
Would it be clearer if I change it and it's 
`($module_available_positions as $value => $module_position)`
or is readability-wise
`($module_positions as $value => $module_position)`
fine too?

Reply:
I would go with `foreach ($optin_form_position_options as $value => $option)`.